client/my-sites/people/people-list-item/index.jsx

```diff
@@ -53,21 +62,40 @@ class PeopleListItem extends React.PureComponent {
 		return type === 'invite' ? inviteLink : editLink;
 	};
 
+	onResend = event => {
+		const { requestingResend, resendSuccess, siteId, inviteKey } = this.props;
+
+		event.preventDefault();
+
+		if ( requestingResend || resendSuccess ) {
+			return null;
+		}
+
+		this.props.resendInvite( siteId, inviteKey );
+	};
+
 	renderInviteStatus = () => {
-		const { invite, translate } = this.props;
+		const { invite, translate, requestingResend, resendSuccess } = this.props;
 		const { isPending } = invite;
-		const statusClasses = {
+		const className = classNames( 'people-list-item__invite-status', {
 			'is-pending': isPending,
-		};
-		const className = classNames( 'people-list-item__invite-status', statusClasses );
+		} );
+		const buttonClassName = classNames( 'people-list-item__invite-resend', {
+			'is-success': resendSuccess,
+		} );
 
 		return (
 			<div className={ className }>
 				{ ! isPending && <Gridicon icon="checkmark" size={ 18 } /> }
 				{ isPending ? translate( 'Pending' ) : translate( 'Accepted' ) }
 				{ isPending && (
-					<Button className="people-list-item__invite-resend" compact>
-						{ translate( 'Resend Invite' ) }
+					<Button
+						className={ buttonClassName }
+						onClick={ this.onResend }
+						busy={ requestingResend }
+						compact
+					>
+						{ resendSuccess ? translate( 'Invite Sent!' ) : translate( 'Resend Invite' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **58** times:
`translate( 'Invite Sent.' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/invites-list-resend)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).